nixnet/_enums.py

```diff
@@ -1093,6 +1093,71 @@ class ObjectClass(enum.Enum):
 
 
```

Comment:
Are the session properties, `types`, `errors`, and `constants` being documented separately?